scripts/yad2_price_scraper.py

```
@@ -0,0 +1,365 @@
+    try:
+        price_elements = driver.find_elements(
+            By.XPATH, "//*[contains(text(), '₪')]"
+        )
+        for el in price_elements:
+            text = el.text.strip()
+            price = extract_price_from_text(text)
+            if price and price > 1000:
+                result["base_price"] = price
+                result["raw_price_text"] = text
+                return result
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)